util/pom.xml

```diff
@@ -100,5 +100,10 @@
       <groupId>com.google.truth</groupId>
       <artifactId>truth</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
`util` project can not depend on Hadoop jars.

Reply:
Deleted.